packages/xstate-solid/test/useMachine.test.tsx

```diff
@@ -32,13 +32,18 @@ afterEach(() => {
 
 describe('useMachine hook', () => {
   const context = {
-    data: undefined
+    data: undefined as string | undefined
   };
-  const fetchMachine = createMachine<
-    typeof context,
-    { type: 'FETCH' } | DoneEventObject
-  >({
+  const fetchMachine = createMachine({
     id: 'fetch',
+    types: {} as {
+      context: typeof context;
+      events: { type: 'FETCH' } | DoneEventObject;
```

Comment:
I think that users shouldn't have to include done events in what they provide for `TEvent`. I've checked this locally and this works without it.
```suggestion
      events: { type: 'FETCH' };
```